docs/GraFx-Studio/guides/template-variables/number/index.md

```diff
@@ -0,0 +1,117 @@
+# Number Variables
+
+## Create a Number Variable
+
+Under the Automate icon, click Variables, and add a variable with the "+" sign.
+
+![screenshot](num0.png)
+
+Choose "Number" as the Variable Type.
+
+![screenshot-full](create-numbers.gif)
+
+## Set the Variable Name
+
+Double-click the name, or choose "Rename" under the "..." menu.
+
+![screenshot](num2.png)
+
+## Set General Properties
+
+![screenshot-full](num1.png)
+
+### Visible
+
+Defines if the variable will be visible in the [Studio UI](/GraFx-Studio/concepts/template-management/#studio-ui) to the end user.
+
+### Range
+
+Defines the acceptable range of values to be accepted by the variable placeholder.
+
+### Decimal Settings
+
+#### Decimal Places
+
+Set the number of decimal places that will be displayed.
+
+#### Decimal Character Style
+
+![screenshot-full](ds0.png)
+
+You can define a character style that is different from the style defined for the text, to use for the decimal part of the number.
+
+The default Paragraph style for the frame is "Text".
+
+![screenshot-full](ds1.png)
+
+A specific character style is defined and has the property to show as "Superscript".
+
+![screenshot-full](ds2.png)
+
+The variable decimal setting shows the decimals will use this character setting for the decimal part.
```

Comment:
Also important to know is the style inheritance model:
- If you want to use the decimal character style, you need to make sure that there are no style overrides on the number variable (you can use the "Clear style overrides" button in the text properties panel to remove the style overrides)
- If a character style is already applied, it is locally replaced by the decimal character style

Reply:
check, added clarification